tests/quality/test_quality.py

```diff
@@ -108,7 +109,7 @@ def get_transformer_regression_scores(data, sdtype, dataset_name, transformers,
                 'column': column,
                 'score': score
             })
-            scores = scores.append(row, ignore_index=True)
+            scores = scores._append(row, ignore_index=True)
```

Comment:
Can we use `pd.concat` here instead of `_append`? I think `pd.concat([scores, row], axis=1])` and then returning `scores.T` at the end should achieve the same result.